src/js/components/Cell/index.js

```diff
@@ -0,0 +1,2 @@
+import Cell from './Cell'
+export default Cell
```

Comment:
eof

Reply:
Adding a specified linter with the spacing would fix your concerns here.  i don't think it's a big deal.  🤷‍♂️   But we can agree on a linter say standard, airbnb or another.

Reply:
The problem is if anyone else automatically adds newlines to the end of files then anytime they view this file it will get updated even though they just viewed the file. I would like to set the standard that all files have a newline at the end.
https://robots.thoughtbot.com/no-newline-at-end-of-file